packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/add/add.html

```diff
@@ -0,0 +1,24 @@
+<oui-modal
+    ng-form="addServiceIntegration"
```

Comment:
I think you missed to create a HTML form element.
Or it's declared at parent template ?

Reply:
As suggested in a previous PR, I removed the form elements when we do not have any submit (this is handled by the oui-modal action attributes. However, we still need the ng-form directive to activate input validation